legacy/src/scss/_patterns_cores-chart.scss

```diff
@@ -70,9 +69,14 @@ $bar-height: 11px;
   }
 
   .p-cores-chart--double-row {
+    top: 3px;
     width: auto;
   }
 
+  .p-cores-chart--overcommit {
```

Comment:
This class is not used. Is it needed?

Reply:
@anthonydillon It's generated dynamically in the `maasCoresChart` directive whenever CPU is overcommitted. It's just the dashed border part.